src/django/api/models/moderation_event.py

```diff
@@ -147,6 +148,16 @@ class ActionType(models.TextChoices):
         )
     )
 
+    action_reason_text_cleaned = models.TextField(
```

Comment:
I suggest to add `max_length` attribute for fields introduced in this PR.

Reply:
The TextField is designed to hold large amounts of text and doesn’t enforce a maximum length at the database level.
https://docs.djangoproject.com/en/5.1/ref/models/fields/#textfield

So, I believe it's not necessary to add a max_length attribute for these fields.

Reply:
Ack.